tests/widgets/drive/drive-scanner.spec.coffee

```diff
@@ -0,0 +1,131 @@
+m = require('mochainon')
+_ = require('lodash')
+Promise = require('bluebird')
+DriveScanner = require('../../../lib/widgets/drive/drive-scanner')
+
+describe 'Drive Scanner:', ->
+
+	describe 'give pre-populated drive options', ->
+
+		beforeEach ->
+			@driveFinder = ->
+				return Promise.resolve([ {foo: 'bar'} ])
+
+		it 'should cause the drives class property to contain the same pre-existing drives', ->
+			scanner = new DriveScanner(@driveFinder, interval: 100, drives: [ {foo: 'bar'} ])
```

Comment:
You're still missing spaces surrounding curly braces in the whole file.

This:

```js
[ {foo: 'bar'} ]
```

Should be:

```js
[ { foo: 'bar' } ]
```